curated_transformers/models/attention.py

```diff
@@ -73,6 +73,16 @@ def apply_causal_mask(input: Tensor) -> Tensor:
     return torch.where(causal_mask, input, blocked_value)
 
 
+class QkvHeadSharing(IntEnum):
+    """Sharing of head parameters."""
+
+    #: No parameters are shared between heads.
+    NONE = 0
+
+    #: Key shares heads, value shares heads, query has separate heads.
```

Comment:
```suggestion
    #: Key and value share heads, query has separate heads.
```

Reply:
I intentionally formulated it like this, otherwise it reads like the key and the value literally use the same head (v = k).

Reply:
Ah, I see.